bitnami/ghost/values.yaml

```diff
@@ -23,128 +63,231 @@ image:
   ## Optionally specify an array of imagePullSecrets.
   ## Secrets must be manually created in the namespace.
   ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## e.g:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
   ##
-  # pullSecrets:
-  #   - myRegistryKeySecretName
+  pullSecrets: []
+  ## Enable debug mode
+  ##
+  debug: false
 
-## Force target Kubernetes version (using Helm capabilites if not set)
+## @section Ghost Configuration parameters
+## Ghost settings based on environment variables
+## ref: https://github.com/bitnami/bitnami-docker-ghost#configuration
+
+## @param ghostUsername Ghost user name
 ##
-kubeVersion:
+ghostUsername: user
+## @param ghostPassword Ghost user password
+## Defaults to a random 10-character alphanumeric string if not set
+##
+ghostPassword: ""
+## @param existingSecret Name of existing secret containing Ghost credentials
+## NOTE: Must contain key `ghost-password`
+## NOTE: When it's set, the `ghostPassword` parameter is ignored
+##
+existingSecret:
+## @param ghostEmail Ghost user email
+##
+ghostEmail: user@example.com
+## @param ghostBlogTitle Ghost Blog title
+##
+ghostBlogTitle: User's Blog
+## @param ghostHost Ghost host to create application URLs
+##
+ghostHost: ""
```

Comment:
I don't see it in the template apart from NOTES.txt

Reply:
It's used in the `_helpers.tpl`